code/game/objects/items/devices/gps.dm

```diff
@@ -13,6 +13,10 @@
 
 /obj/item/gps/Initialize()
 	. = ..()
+	_add_gps_component()
+
+/// Adds the GPS component to this item.
+/obj/item/gps/proc/_add_gps_component()
```

Comment:
Why the `_` before?

Reply:
Python way of naming functions to indicate "private" function, since this shouldn't really be called by anything outside the item, it's just used for overriding in child classes.

Can change it if you object?

Reply:
I've seen this pattern for unused variables, but wasn't aware of the private functions one.

I'd say the best would be to stay consistent, so change it for now and propose adding it to the contribution guidelines if you'd like for it to become a codebase standard.
I don't have strong opinions on it, but I'd really not like to see a million mixed standards proliferating.